src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java

```diff
@@ -34,12 +35,42 @@
  * Calculate read-counts across targets given their reference coordinates.
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
+ *     You may obtain better results with different parameters.
+ * </p>
+ *
+ * <p>For whole exome sequencing (WES) data: </p>
+ *
+ * <pre>
+ * java -Xmx4g -jar $gatk_jar CalculateTargetCoverage \
+ *   --input sample.bam \
+ *   --targets padded_targets.tsv \
+ *   --groupBy SAMPLE \
+ *   --transform PCOV \
+ *   --targetInformationColumns FULL \
+ *   --disableReadFilter NotDuplicateReadFilter \
```

Comment:
Because the defaults were changed so that they could be removed. Defaults are now changed back and these parameters have to be put back in.